src/ast/passes/resolve_imports.cpp

```diff
@@ -340,15 +308,15 @@ Result<OK> ResolveImports::importAny(Node &node, const std::string &name)
 void ResolveImports::visit(Import &imp)
 {
   // Ensure that the explicit statement is allowed.
-  if (bpftrace_.config_->unstable_import == ConfigUnstable::error) {
+  if (enabled_ == ConfigUnstable::error) {
```

Comment:
Do we need to do this check here rather than just in `UnstableFeature` class?

Reply:
Let me check, and I'll remove this if it's not needed. That would be great.